pkg/version/version.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 The nebula-contrib Authors.
+ * Licensed under the Apache License, GitVersion 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package version
+
+var (
+	GitVersion = "v0.0.0-main"
+)
+
+func GetVersion() string {
+	// 打印当前包信息
+
+	return GitVersion
+}
```

Comment:
distinguish between ngctlVersion and GitRef is better? we can get dynamically get GitRef and GitSha , then use fmt.Sprintf to format them